pom.xml

```diff
@@ -1471,7 +1471,7 @@
     <protobuf.plugin.version>0.5.0</protobuf.plugin.version>
     <thrift.path>thrift</thrift.path>
     <thrift.version>0.12.0</thrift.version>
-    <zookeeper.version>3.4.10</zookeeper.version>
+    <zookeeper.version>3.5.5</zookeeper.version>
     <!-- What ZooKeeper 3.4.x depends on and nothing more -->
```

Comment:
this comment was also added by https://issues.apache.org/jira/browse/HBASE-20108
we can update it to "What ZooKeeper 3.5.5 depends on...", but I think it is a bit hard to decode this comment and I would add more info, like: "providing JLine on the classpath for zookeeper (see HBASE-20108)"